### PR TITLE
Bugfix 3.4:  Clean up rocksdb getStatistics()

### PR DESCRIPTION
**Pull request by matthewvon**
The statistics from rocksdb were unknowingly focused upon "default" column family and therefore excluding values from the seven column families used by ArangoDB.  This PR updates several values such that they report the sum across the seven column families.  The corrected statistics are essential to performance measurement.

The current value of the RocksDBThrottle is a new statistic added in this PR.

Upon merge, the same code will go to devel, 3.5.x, and maybe 3.3 branches.

### CODE REVIEW

**Comment by matthewvon** [comment]
Running ... http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/5678/

File: arangod/RocksDBEngine/RocksDBEngine.cpp [hunk 1]
```diff
@@ -2121,6 +2121,23 @@ void RocksDBEngine::getStatistics(VPackBuilder& builder) const {
     }
   };
 
+  // get string property from each column family and return sum;
+  auto addStrAllCf = [&](std::string const& s) {
```
Comment (danielhlarkin) [inline arangod/RocksDBEngine/RocksDBEngine.cpp]: Wouldn't this be `addIntAllCf`?

Comment (matthewvon) [reply arangod/RocksDBEngine/RocksDBEngine.cpp]: Open to changing.  I was focused at the time on what the rocksdb GetProperty call was using as a return type.  All ColumnFamily calls return std::string.  Yes, I convert its returned strings to int.  Mostly upset that anyone thought returning a number as a string was the correct action.

Comment (danielhlarkin) [reply arangod/RocksDBEngine/RocksDBEngine.cpp]: Agreed.

**Pull request merged**